coldfront/plugins/qumulo/utils/qumulo_api.py

```diff
@@ -203,15 +203,14 @@ def list_nfs_exports(self):
 
     def update_allocation(
         self,
-        protocols: list[str],
+        protocols: Optional[list[str]],
         export_path: str = None,
         fs_path: str = None,
         name: str = None,
         limit_in_bytes: int = 0,
     ):
-        if not protocols:
-            raise ValueError("protocols should be defined.")
-
+        if protocols is None:
```

Comment:
Since the method signature has default parameter values, isn't setting a default for the protocols appropriate?

```
    def update_allocation(
        self,
        protocols: list[str] = [],
        export_path: str = None,
        fs_path: str = None,
        name: str = None,
        limit_in_bytes: int = 0,
    ):
```